Jenkinsfile_CNP

```diff
@@ -122,6 +122,12 @@ withPipeline(type, product, component) {
     """
     }
 
+    afterSuccess('akschartsinstall') {
+        onPR {
+            githubUpdateDeploymentStatus(githubCreateDeployment(), "https://ia-case-api-pr-${CHANGE_ID}-aip-frontend.preview.platform.hmcts.net")
+            githubUpdateDeploymentStatus(githubCreateDeployment("-PR-${CHANGE_ID}"), "https://xui-ia-case-api-pr-${CHANGE_ID}." + "preview.platform.hmcts.net")
```

Comment:
No need for the PR reference here. On the other hand, it would be useful to have a reference to the journey. Change it as follows:

```suggestion
githubUpdateDeploymentStatus(githubCreateDeployment("-LR"), "https://xui-ia-case-api-pr-${CHANGE_ID}." + "preview.platform.hmcts.net")
```